### PR TITLE
Tech writing review of Minikube guide

### PR DESCRIPTION
**Pull request by sarahmaddox**
Did a quick pass of the guide to apply the Kubeflow docs style guide, and linked from the Linux guide to the Minikube guide.

Preview: https://deploy-preview-1921--competent-brattain-de2d6d.netlify.app/docs/started/workstation/minikube-linux/

### CODE REVIEW

**Comment by sarahmaddox** [comment]
/assign @janeman98 

@janeman98 Please would you take a look at this PR?

File: content/en/docs/started/workstation/getting-started-linux.md [hunk 1]
```diff
@@ -50,7 +50,11 @@ page will complete this path.
...
-Similar to the Kubeflow appliance, the Kubernetes appliance is a virtual machine has a
-Kubernetes cluster already installed. After starting the virtual machine you will need
+A Kubernetes appliance is a virtual machine that has a
+Kubernetes cluster already installed. After starting the virtual machine, you need
 to install Kubeflow. This option gives you full control over your Kubeflow setup.
 
+- Install [Minikube](https://kubernetes.io/docs/setup/learning-environment/minikube/)
+
+Follow the instructions on [deploying with MiniKube on 
+Linux](/docs/started/workstation/minikube/linux/) to complete this path.
```
Comment (janeman98) [inline content/en/docs/started/workstation/getting-started-linux.md:60]: Thanks @sarahmaddox ! 
Just a minor change: 

The Link should be 
/docs/started/workstation/minikube-linux/

Other option is to change the name of Minikube tutorial.

Comment (sarahmaddox) [reply content/en/docs/started/workstation/getting-started-linux.md]: Oh well spotted! Fixed.

**Comment by janeman98** [comment]
Thanks @sarahmaddox !
/lgtm
/approve

**Pull request merged**